CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+v1.2.0
```

Comment:
I would expect this to be v2.0.0 since the `sbvr-compiler.coffee` API surface changes to an object with methods, instead of a single exported method.

Reply:
I made it backwards compatible